### DISCUSSION TITLE
Problem with Release v1.8.0

### DISCUSSION THREAD

**Discussion by JLD95** [maartentamboer/X-Touch-Mini-FS2020#82] [Q&A] [answered]
Hi,
the previous version works fine but Ihe v1.8.0 don't work with my configuration: only a windows appears/disappears very quickly.  I use the same <config_a320.json> for the last two versions. Any idea ?

PS: in cmd windows, I have this error

 Traceback (most recent call last):
  File "main.py", line 4, in <module>
ModuleNotFoundError: No module named 'MSFSPythonSimConnectMobiFlightExtension'
[1988] Failed to execute script main

**Comment by maartentamboer** [comment]
Is release 1.7.0 working for you? https://github.com/maartentamboer/X-Touch-Mini-FS2020/releases/tag/v1.7.0

**Reply by maartentamboer** [reply]
Can you try this attached zip from #83 
[X-Touch-Mini-FS2020-83.zip](https://github.com/maartentamboer/X-Touch-Mini-FS2020/files/7007497/X-Touch-Mini-FS2020-83.zip)

**Comment by JLD95** [comment]
Yes v1.7.0 and your X-Touch-Mini-FS2020-83.zip works fine

**Comment by maartentamboer** [comment, marked as answer]
v1.8.1 has been released and fixes this issue
